playground/mocks/data/idp/idx/authenticator-verification-custom-otp.json

```diff
@@ -0,0 +1,184 @@
+{
+  "stateHandle": "02UxxOzemJmZMFgNfWePmTbaWZvWyJyr-7hi_ps4Iu",
+  "version": "1.0.0",
+  "expiresAt": "2021-01-04T03:33:45.000Z",
+  "intent": "LOGIN",
+  "remediation": {
+    "type": "array",
+    "value": [
+      {
+        "rel": [
+          "create-form"
+        ],
+        "name": "challenge-authenticator",
+        "relatesTo": [
+          "$.currentAuthenticator"
+        ],
+        "href": "http://localhost:3000/idp/idx/challenge/answer",
+        "method": "POST",
+        "produces": "application/ion+json; okta-version=1.0.0",
+        "value": [
+          {
+            "name": "credentials",
+            "type": "object",
+            "form": {
+              "value": [
+                {
+                  "name": "otp",
```

Comment:
Per discussion with @srinivasanagandla-okta yesterday, RSA, Google, and Custom OTP should all also be using `credentials.passcode` instead of `credentials.otp` in order to align with email/phone authenticators.

FYI @TianGan-okta @jasleennuno-okta @ManuMalhotra-okta @valberbenetz-okta